cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfParser.h

```diff
@@ -100,6 +100,7 @@ struct UrdfLinkDesc
 
     // derived data
     bool has_parent_ = false;
+    UrdfJointDesc* parent_joint_desc_ = nullptr;
     std::vector<UrdfLinkDesc*> child_link_descs_;
     std::vector<UrdfJointDesc*> child_joint_descs_;
```

Comment:
- if we're storing `parent_joint_desc_`, then we don't also need `child_joint_descs_`, so we can remove it